dvc/fs/http.py

```diff
@@ -114,7 +123,7 @@ async def get_client(self, **kwargs):
             total=None,
             connect=self.REQUEST_TIMEOUT,
             sock_connect=self.REQUEST_TIMEOUT,
-            sock_read=None,
+            sock_read=self.REQUEST_TIMEOUT,
```

Comment:
I am curious to see if only this will solve the problem in `dvc-bench`, and a bit hesitant on adding `TCPConnector`
without trying out this solution first. Would you mind opening another PR with just this line change? We can merge that, and we can keep this PR open for a 1-2 days to monitor.

Reply:
I've tried, it solves the issues on my machine :tm: (MacOS) but it still freezes on the `dvc-bench` ci